README.md

```diff
@@ -168,6 +168,8 @@ Many of these parameters are only used in the `:enable` action.
 - **restart_on_update** - Whether the service should be restarted when
     the run script is updated. Defaults to `true`. Set to `false` if
     the service shouldn't be restarted when the run script is updated.
+- **down** - Set the default state of the runit service to 'down' by creating
```

Comment:
n/m, I think `start_down` is two things: 1) clear enough if you're familiar with the functionality and 2) confusing enough that if you're not familiar with the functionality you probably won't make an uneducated assumption and actually take the time to look it up.  I'll go with `start_down`